data_base/schema.sql

```diff
@@ -0,0 +1,20 @@
+CREATE DATABASE catalog;
+
+CREATE TABLE books (
+id SERIAL PRIMARY KEY,
+	publisher VARCHAR(250),
+	cover_state VARCHAR(250),
+    publish_date DATE,
+    archived BOOLEAN
```

Comment:
@monicah79, Excellent 👍  work on creating the schema tables for the classes. As a recommendation, please ensure that unnecessary and extra spacing is removed and that your document is formatted according to best practices before committing any changes.